Sources/PT.PM.Matching/Patterns/PatternInvocationExpression.cs

```diff
@@ -41,7 +49,15 @@ public override MatchContext Match(InvocationExpression invocation, MatchContext
             {
                 return newContext;
             }
-            newContext = Arguments.Match(invocation.Arguments, newContext);
+
+            if (Arguments is PatternArgs patternArgs)
+            {
+                newContext = patternArgs.Match(invocation.Arguments, newContext);
+            }
+            else
+            {
+                newContext = context.Fail();
```

Comment:
I suggest change this string to `ewContext = Arguments.MatchUst(invocation.Arguments, newContext);`. It will  work for or logical operation.

Reply:
add this in last commit